src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,16 @@
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
you don't need another font-face, just import it and use (if does not work via styles, add via HTML)